dat/missions/neutral/cargo_rush.lua

```diff
@@ -145,6 +145,9 @@ function accept()
          misn.finish()
       end
    end
+   misn.createLog( buildCargoMissionDescription(cargosize[tier], amount, cargo, destplanet, destsys ), "Trade",0)
+   origin_p, origin_s = planet.cur()
+   misn.appendLog("Cargo accepted on "..origin_p:name() .. " for " .. destplanet:name())
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.